README.md

```diff
@@ -27,7 +27,7 @@ Sometimes this is just a more practical and quick way than doing things properly
 
 ### Generate random 6-hexdigit URL:
 
-    miniserve --random-route -i 192.168.0.1 /tmp
+    miniserve --random-route -i 192.168.0.1 -- /tmp
```

Comment:
Aw, this is ugly, can we get rid of the `--` in this case somehow?

Reply:
It should be possible as their example has a similar case and they don't need `--`: https://github.com/TeXitoi/structopt#example